usecases/backup/backend.go

```diff
@@ -121,21 +124,21 @@ func (s *objStore) meta(ctx context.Context, key string, dest interface{}) error
 	return nil
 }
 
-type nodeStore struct {
-	objStore
+type NodeStore struct {
```

Comment:
do we need to have this struct public now?

Reply:
Yes